test/AdobeState.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/no-conditional-expect */
```

Comment:
instead of disabling, I would restructure the code as recommended by Jest:
https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-conditional-expect.md#how-to-catch-a-thrown-error-for-testing-without-violating-this-rule

I did it in the app plugin:
https://github.com/adobe/aio-cli-plugin-app/commit/57beafe349353d803a33133d03ebf2efc0fd0155

Reply:
true thanks!